### PR TITLE
testing telegram preview

### PR DESCRIPTION
**Pull request by supermaven**

### CODE REVIEW

File: front/src/pages/SharePage/SharePage.tsx [hunk 1]
```diff
@@ -46,9 +46,9 @@ const SharePage = ({ history }) => {
               <Row style={{ padding: '20px' }}>
                 <div className='preview-area center'>
                   <Preview
-                    url={file.url}
+                    url={file.thumbnail}
                     ext={file.fileExtension}
-                    type={file.mimeType}
+                    type='image/png'
```
Comment (mihaiiorga) [inline front/src/pages/SharePage/SharePage.tsx:51]: I don't think this is a good solution, what if it's a jpg image or any other kind of file?

Comment (supermaven) [reply front/src/pages/SharePage/SharePage.tsx:51]: well, all thumbnail image type is png , right?

File: templates/shortlink.html [hunk 2]
```diff
@@ -11,14 +11,16 @@
...
+  <meta itemprop="image" content="https://www.opacity.io/logo.png">
+  
 
   <!-- Facebook Meta Tags -->
   <meta property="og:url" content="{{.Shortlink}}">
   <meta property="og:type" content="website">
   <meta property="og:title" content="Opacity.io">
-  <meta property="og:image" content={{.Image}}>
-  <meta property="og:image:secure_url" content={{.Image}} />
+  <meta property="og:image" content="https://www.opacity.io/logo.png">
```
Comment (mihaiiorga) [inline templates/shortlink.html:22]: according to this: https://ogp.me/#metadata the content tag should have the actual value of the URL of the file

File: front/src/pages/SharePage/SharePage.tsx (hunk 1 above)
Comment (mihaiiorga) [reply front/src/pages/SharePage/SharePage.tsx:51]: no, they weren't. However, all of them are now

**Review by mihaiiorga — APPROVED** [review]

**Pull request merged**